device/simulation/tt_simulation_device_generated.h

```diff
@@ -0,0 +1,215 @@
+// automatically generated by the FlatBuffers compiler, do not modify
+
+
+#ifndef FLATBUFFERS_GENERATED_TTSIMULATIONDEVICE_H_
+#define FLATBUFFERS_GENERATED_TTSIMULATIONDEVICE_H_
+
+#include "flatbuffers/flatbuffers.h"
+
+// Ensure the included flatbuffers.h is the same version as when this file was
+// generated, otherwise it may not be compatible.
+static_assert(FLATBUFFERS_VERSION_MAJOR == 24 &&
+              FLATBUFFERS_VERSION_MINOR == 3 &&
+              FLATBUFFERS_VERSION_REVISION == 25,
+             "Non-compatible flatbuffers version included");
+
+struct tt_core;
```

Comment:
Is there a way to namespace the autogenerated code to avoid the possibility of collisions?  I ask because `tt_core` is somewhat generic.

Reply:
I don't think so, but I can change the name of the struct to something like `tt_simulation_core` and regenerate the flatbuffer schema

Reply:
changed to `tt_vcs_core`